mne_bids/config.py

```diff
@@ -2,20 +2,6 @@
 from mne import io
 from mne.io.constants import FIFF
 
-try:
-    from mne.io import read_raw_persyst
-except ImportError:
```

Comment:
Can this really go or did you delete it by accident?

Reply:
Well if mne-bids 0.5 is using mne-python 0.21+ now, then it can go?

Reply:
Yes, ok. Just wanted to double-check :)